### DISCUSSION TITLE
Sponsorship with several instances

### DISCUSSION THREAD

**Discussion by fernandonajera** [photoprism/photoprism#2862] [Q&A] [answered]
I cannot find the answer to this question anywhere: if I become a sponsor and I run two/a few instances in parallel, do I get the benefits in all of them?

**Comment by lastzero** [comment, marked as answer]
Right now, it does not make a difference, unless you sell them as a web hoster. In the future, we may limit the number of instances based on your sponsorship level.

**Reply by fernandonajera** [reply]
Alright, thank you! I’m definitely not going to resell anything, but I would like to keep two or three sets of pictures completely separated.

**Reply by lastzero** [reply]
We don't have any specific plans yet, but I think it makes sense to allow (at least) 3 instances for [PhotoPrism+ Members](https://photoprism.app/membership). Feedback from (potential) users is very valuable to figure out the details. Thanks for that and I hope you enjoy our software!
